src/QueryGenerators.js

```diff
@@ -346,6 +358,21 @@ class DefaultQueryGenerator {
 		} ;
 	}
 
+	initFilterSearch(texte, variable) {			
```

Comment:
I think this isn't necessary as it duplicates an already existing method. Any reason for this ?
Otherwise could you update the PR without this method ? Thanks

Reply:
oops copy past error, I'll fix it